Actualizacion de valores de region

dany-herz42 opened the pull request:
## Cambios en el archivo initializers/decidim.rb
- **Actualización del tipo de moneda del sistema**: 
  - El valor de config.currency_unit cambio de "€" a "$".
- **Actualización del idioma disponible para el sistema:**
  - El valor de config.available_locales cambio a [:"es-MX"]
  - El valor de config.default_locale cambio a :"es-MX"

Co-authored-by: corps01 <ocorpus@ucol.mx>

dany-herz42 committed:
Actulizacion de valores de region

-Actualizacion del tipo de moneda del sistema.
-Actualizacion del idioma disponible para el sistema

Co-authored-by: corps01 <ocorpus@ucol.mx>

oxcar commented:
@EduardoGHdez había comentado de poner en idiomas disponibles también el "es"para que quedara de fallback en caso de que no estén todos los textos traducidos.

oxcar commented on config/initializers/decidim.rb:
```diff
@@ -12,12 +12,12 @@
   # be able to choose the available languages for that organization. That list
   # of languages will be equal or a subset of the list in this file.
-  config.available_locales = [:en, :ca, :es]
+  config.available_locales = [:"es-MX"]
```
Aquí, poner el de español de Eshpaña. "es"

oxcar replied:
Añadir... para que estén ambas.

dany-herz42 committed:
Se agregó al idioma "es" en available_locales

- **Cambios en config.available_locales: **
  - Se agregó el idioma castellano a la configuracion de idiomas disponibles

oxcar reviewed (commented):
Me parece correcto.
